app/src/main/res/values/strings.xml

```diff
@@ -306,9 +316,29 @@ Please report any problems you encounter via Github.</string>
     <string name="sync_only_wifi_ssids_location_permission_rejected_dialog_title">Permission required</string>
     <string name="sync_only_wifi_ssids_location_permission_rejected_dialog_content">Starting with Android 8.1, location access is required to be able to read the WiFi\'s name. You can use this feature only if you grant this permission.</string>
 
+    <string name="power_source_title">Run when device powered by</string>
+
+    <string-array name="power_source_entries">
+        <item>AC and battery power</item>
+        <item>AC power</item>
+        <item>Battery power</item>
```

Comment:
Consequently you need to add a final dot here as well, if you do it on line 328 below.